src/Stratis.SmartContracts.CLR/CallDataSerializer.cs

```diff
@@ -163,6 +166,18 @@ protected void AddMethodParams(List<byte[]> rlpBytes, object[] methodParameters)
             }
         }
 
+        protected void AddSignatures(List<byte[]> rlpBytes, string[] signatures)
+        {
+            if (signatures != null && signatures.Any())
```

Comment:
Perhaps `Guard` null if the signatures is null or empty? Then remove the null and `Any` checks?

Reply:
Done.